packages/tokens/src/types.ts

```diff
@@ -638,7 +640,6 @@ export type ShadowBrandTokens = {
 };
 
 export type Greys =
-  | 0
```

Comment:
The two shades of grey have been removed by the token pipeline. Agreed on async, those are not exported (global colors) and never used (As Grey.0 is Black and Grey.100 is White).